modules/andes-core/broker/src/main/java/org/wso2/andes/server/queue/IncomingMessage.java

```diff
@@ -91,33 +91,12 @@ public void setContentHeaderBody(final ContentHeaderBody contentHeaderBody) thro
         _contentHeaderBody = contentHeaderBody;
     }
 
+    /**
+     * Set the time to live value for the incoming messages
+     */
     public void setExpiration()
     {
-            long expiration =
-                    ((BasicContentHeaderProperties) _contentHeaderBody.getProperties()).getExpiration();
-            long timestamp =
-                    ((BasicContentHeaderProperties) _contentHeaderBody.getProperties()).getTimestamp();
-            if (SYNCED_CLOCKS)
-            {
-                _expiration = expiration;
-            }
-            else
-            {
-                // Update TTL to be in broker time.
-                if (expiration != 0L)
-                {
-                    if (timestamp != 0L)
-                    {
-                        // todo perhaps use arrival time
-                        long diff = (System.currentTimeMillis() - timestamp);
-
-                        if ((diff > 1000L) || (diff < 1000L))
-                        {
-                            _expiration = expiration + diff;
-                        }
-                    }
-                }
-            }
+        _expiration = ((BasicContentHeaderProperties) _contentHeaderBody.getProperties()).getExpiration();
```

Comment:
this doesn't conform to our standards. however, this convention is already there in the code we inherited from qpid. 
lets keep it like this until we revamp transports